bluesky/tests/test_callbacks.py

```diff
@@ -573,3 +574,21 @@ def event(self, doc):
                              reg=db.fs)
     RE.subscribe(bc)
     RE(count([det]))
+
+
+def test_broker_base_no_unpack(fresh_RE, db):
+    class BrokerChecker(BrokerCallbackBase):
+        def __init__(self, field, *, db=None):
+            super().__init__(field, db=db)
+
+        def event(self, doc):
+            super().event(doc)
+            assert isinstance(doc['data'][self.fields[0]], np.ndarray)
+
+    RE = fresh_RE
+    RE.subscribe(db.insert)
```

Comment:
I would remove this (and the `db` fixture altogether). `BrokerCallbackBase` doesn't need it, right? This would simplify the test and also fix the sqlite + numpy array failure that killed Travis.

Reply:
I just took the other approach using the `raises` flag in `xfail` (although I could do this too, but I applied that one to the previous callback base test, which should make that one more robust)

Reply:
Fair enough. This also needs a `@py3` decorator.